### PR TITLE
More Spanish fixes

### PR DESCRIPTION
**Pull request by emilianocaro**
My first pull req :P I will keep reviewing the Spanish translation, until it's finished.

### CODE REVIEW

**Review by mperugini — APPROVED** [review]
Buen PR Emi!!

**Comment by emilianocaro** [comment]
jajaja que haces aca?

File: application/language/es/common_lang.php [hunk 1]
```diff
@@ -28,7 +28,6 @@
 $lang["common_gender_female"] = "F";
 $lang["common_gender_male"] = "M";
 $lang["common_gender_undefined"] = "";
-$lang["common_icon"] = "";
```
Comment (jekkos) [inline application/language/es/common_lang.php:31]: this should probably remain here

File: application/language/es/common_lang.php [hunk 2]
```diff
@@ -64,16 +61,14 @@
...
 $lang["common_search_options"] = "Opciones de búsqueda";
 $lang["common_searched_for"] = "Buscado";
-$lang["common_software_short"] = "";
-$lang["common_software_title"] = "";
 $lang["common_state"] = "Estado";
 $lang["common_submit"] = "Enviar";
 $lang["common_total_spent"] = "Total gastado";
 $lang["common_unknown"] = "Desconocido";
 $lang["common_view_recent_sales"] = "Ver Ventas Recientes";
-$lang["common_website"] = "opensourcepos.org";
```
Comment (jekkos) [inline application/language/es/common_lang.php:74]: not sure if this should be changes, it might show in the footer, I'd leave it as it was

**Comment by jekkos** [comment]
@emilianocaro check my PR first it solves the comments I added here. I would like to give you the credits so if you can submit that version or fix the comments here then I will merge it.

You can also use weblate and that will push the translations directly when done.

**Comment by emilianocaro** [comment]
mmmm wierd. I dind't modified those lines you mention. I don't know what happened there. I'll check that file again.

**Comment by emilianocaro** [comment]
Ok, now files should be ok, I reviewd one by one. Added "Debit card" filter, I'm working with git fork software, and I think that's the one changing my empty labels. I had to add a lot of text again to the repository files 🤦‍♂️.

**Comment by jekkos** [comment]
@emilianocaro I prefer to have one PR per changeset, in this case it would be cleaner to create two PRs

* one for language updates
* one for the debit filter

Also try to avoid adding merge commits in your pull request as it pollutes the master branch history

**Comment by jekkos** [comment]
So it might be easier to go one by one if you're not very familiar with using git yet. Start from the latest master in your fork and apply the language changes. After we merge that then you can pull in master again, and apply the second changeset and create a PR again.

**Comment by emilianocaro** [comment]
Actually no, it's not easier for me to go one by one. I won't spend my time in learning github. I brought to you some error fixes in spanish language, and some things that were missing. If that helps u good, but I'm not gonna waste my time in learning the whole code, and how does github work just to make it "less pollute" for you.

**Comment by jekkos** [comment]
Ok then why would I waste my time and try to help you at all? Doesn't give me any benefit either

**Comment by jekkos** [comment]
I shouldn't even be wasting my time answering this reply..

**Pull request closed without merging**